rfcs/0013-avalanche.md

```diff
@@ -0,0 +1,98 @@
+# Replication via Avalanche 
+
+This RFC describes the current single layer replication mechanism as well as proposed changes to add a second layer and implementing an Avalanche mechanism.
+
+## Current Design
+There's two basic parts to the current Replication design. 
+
+#### Broadcast stage:
```

Comment:
No need for colons after headers. Hit the "View" button on this review to see the rendered markdown.